src/Authenticator/JwtAuthenticator.php

```diff
@@ -166,6 +168,24 @@ protected function decodeToken(string $token): ?object
             );
         }
 
+        $jsonWebKeySet = $this->getConfig('jsonWebKeySet');
```

Comment:
This config should also be added to `$_defaultConfig` array with default `null` value.

I am also wondering if the config name should be shortened to `jwks`. Lesser chances of making a typo with a shorter name plus most webservice docs would be using the term `JWKS` so recognising what this option is for shouldn't be an issue.

Reply:
sorry - seem to have missed your latest review - made the required changes - do you want me to change the name of the variable as well or is it enough to change the config key?

Reply:
The variable is local and users don't need to care about it, so it's fine.